### PR TITLE
New keyboard SLZ40

### PR DESCRIPTION
**Pull request by sithlord121**
## Description

New keyboard, 40% style keyboard with 11 num row keys on top row for Zhuyin input.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by sithlord121** [comment]
updated info.json file

File: keyboards/slz40/keymaps/default/keymap.c [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    _FN,
+	_FN2
+};
+
+// Defines the keycodes used by our macros in process_record_user
+enum custom_keycodes {
+    PSWD1 = SAFE_RANGE,
+    QMKURL
+};
+
```
Comment (zvecr) [inline keyboards/slz40/keymaps/default/keymap.c:30]:
```suggestion
```

File: keyboards/slz40/keymaps/default/keymap.c [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+            if (record->event.pressed) {
+                // when keycode QMKURL is pressed
+                SEND_STRING("https://qmk.fm/\n");
+            } else {
+                // when keycode QMKURL is released
+            }
+            break;
+    }
+    return true;
+}
```
Comment (zvecr) [inline keyboards/slz40/keymaps/default/keymap.c:76]:
```suggestion
```

**Review by zvecr — CHANGES_REQUESTED** [review]

File: keyboards/slz40/readme.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# slz40
+
+![slz40](imgur.com image replace me!)
```
Comment (fauxpark) [inline keyboards/slz40/readme.md:3]: Please fill this out.

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/slz40/readme.md (hunk 3 above)
Comment (sithlord121) [reply keyboards/slz40/readme.md]: updated

**Review by zvecr — APPROVED** [review]

**Review by drashna — APPROVED** [review]

**Review by fauxpark — APPROVED** [review]

**Pull request merged**